core/src/main/java/tc/oc/pgm/events/MapVoteWinnerEvent.java

```diff
@@ -0,0 +1,29 @@
+package tc.oc.pgm.events;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import tc.oc.pgm.api.map.MapInfo;
+
+public class MapVoteWinnerEvent extends Event {
```

Comment:
Why is this event needed? Also, could this be a `MatchEvent`?

Reply:
Probably used to return the token if your sponsored map wins the vote